cli/commands/commands.go

```diff
@@ -42,14 +43,17 @@ import (
 // AddCommands adds all subcommands
 func AddCommands(cmd *cobra.Command) {
 	cmd.AddCommand(
+		// commands for resources
 		login.NewLoginCommand(),
```

Comment:
Seems that the package name always is included in the function names as well. In general, such repetition should be avoided according to this [go dev blog](https://go.dev/blog/package-names) (at least for types).
But in that case, I guess it is ok, since 1st) just `New` or `NewCommand` is not meaningful enough either and 2nd) replace with a generic name like `NewParentCommand` is also not that meaningful.

Reply:
In this case, this seems to be somewhat idiomatic to the cobra/viper package and as far as I remember they structure the commands according to this schema in their examples, so I would stick with this.